presto-docs/src/main/sphinx/sql/show-stats.rst

```diff
@@ -0,0 +1,31 @@
+==========
+SHOW STATS
+==========
+
+Synopsis
+--------
+
+.. code-block:: none
+
+    SHOW STATS FOR table
+    SHOW STATS FOR ( SELECT * FROM table [ WHERE condition ] )
+
+Description
+-----------
+
+Returns stats for the named table or for the results of a (limited) query.
+
+Stats are returned for each column, along with a rollup across all columns
+(the column will be ``NULL`` for this rollup).
+
+=====================   ============
+Column                  Description
+=====================   ============
+column_name             The name of the column
+data_size               The total size in bytes of all of the values in the column
+distinct_values_count   The approximate number of distinct values in the column
+nulls_fractions         The portion of the values in the column that are ``NULL``
+row_count               The number of rows  (only returned for the rollup column)
+low_value               The lowest value found in this column (only for numeric types)
```

Comment:
is DATE numeric? do we show low/high for DATEs? 

do we show low/high value for VARCHAR? (we used to, but i am not sure of current state)

Reply:
I just checked using `HiveQueryRunner` and do show min/max for dates. Maybe change "numeric" to "some"